src/main/java/net/sf/jabref/logic/exporter/BibtexDatabaseWriter.java

```diff
@@ -109,7 +113,10 @@ protected void writeString(BibtexString bibtexString, boolean isFirstString, int
                 getWriter().write("{}");
             } else {
                 try {
-                    String formatted = new LatexFieldFormatter().format(bibtexString.getContent(), LatexFieldFormatter.BIBTEX_STRING);
+                    String formatted = new LatexFieldFormatter(
+                            LatexFieldFormatterPreferences.fromPreferences(jabRefPreferences))
```

Comment:
Could you include the `LatexFieldFormatterPreferences` in `SavePreferences` (and remove the explicit reference on `JabRefPreferences` from `BibtexDatabaseWriter`)